main.go

```diff
@@ -19,22 +22,29 @@ var (
 )
 
 func main() {
+	err := mainWithError()
+	if err != nil {
+		panic(fmt.Sprintf("%#v\n", microerror.Mask(err)))
+	}
+}
+
+func mainWithError() error {
```

Comment:
This is something I would like to do in other projects as well. It is useful to directly see the error stack in the panic message.